console/network/src/consensus_heights.rs

```diff
@@ -87,7 +90,7 @@ impl std::fmt::Display for ConsensusVersion {
 }
 
 /// The number of consensus versions.
-pub(crate) const NUM_CONSENSUS_VERSIONS: usize = 11;
+pub(crate) const NUM_CONSENSUS_VERSIONS: usize = 12;
```

Comment:
We can avoid this magic number with the following change.
```suggestion
pub(crate) const NUM_CONSENSUS_VERSIONS: usize = enum_iterator::cardinality::<ConsensusVersion>();
```